en/developers-guide/proof-of-reserve.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Ongoing proof of reserve validation for your financial entity.
+---
+
+# Proof of Reserve
+
+For a comprehensive illustration of utilizing **Orakl Network's Proof of Reserve**, refer to the 
+example repository of the [`data-feed-consumer`](https://github.com/Bisonai/data-feed-consumer). The on-chain implementation for Proof of Reserve mirrors that of the Aggregator on the Aggregator Proxy.
+
+## What is Proof of Reserve?
+
+The **Orakl Network's Proof of Reserve (PoR)** stands as a vital element in fostering trust and transparency within the financial ecosystem of 
+Orakl Network. Playing a pivotal role, Proof of Reserve is dedicated to the verification and validation of reserve holdings for financial 
```

Comment:
Please do not mention "verification". PoR does not verify anything.

Reply:
At https://orakl.network/proof-of-reserve, we call it `GPC (Gold Pegged Coin)` and in here here we use `Peg-Gold` and `PEG Gold`. Please unify it according to the website if there is no reason behind using different names in here.